app/(auth)/login/page.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+
+import { useRef } from 'react';
+import { signIn } from 'next-auth/react';
+import Image from 'next/image';
+// components
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+import { Label } from '@/components/ui/label';
+
+// ----------------------------------------------------------------
+
+const Login = () => {
+  const email = useRef('');
+  const password = useRef('');
```

Comment:
We should be using controlled components everywhere, not refs for reading the values.